src/main/java/org/sagebionetworks/bridge/services/ParticipantFileService.java

```diff
@@ -163,12 +150,14 @@ private String getFilePath(ParticipantFile file) {
     }
 
     private URL generatePresignedRequest(ParticipantFile file, HttpMethod method) {
-        Date expiration = new DateTime().plusMinutes(EXPIRATION_IN_MINUTES).toDate();
+        DateTime expiration = file.getCreatedOn().plusMinutes(EXPIRATION_IN_MINUTES);
+
         GeneratePresignedUrlRequest request = new GeneratePresignedUrlRequest(bucketName, getFilePath(file), method);
-        request.setExpiration(expiration);
+        request.setExpiration(expiration.toDate());
         if (PUT.equals(method)) {
             request.setContentType(file.getMimeType());
             request.addRequestParameter(Headers.SERVER_SIDE_ENCRYPTION, ObjectMetadata.AES_256_SERVER_SIDE_ENCRYPTION);
+            file.setExpires(expiration);
```

Comment:
Pre-signed URLs expire for both get and put. We should put in the expiration date regardless.